crates/goose-mcp/src/google_drive/mod.rs

```diff
@@ -36,6 +36,8 @@ use google_drive3::{
 use google_sheets4::{self, Sheets};
 use http_body_util::BodyExt;
 
+const GOOGLE_DRIVE_SCOPES: Scope = Scope::Full;
```

Comment:
any worry about deletions or unwanted consequences from changing this to full (which I assume is read and manage everything?). Guess this also depends on what you're enabling with the tools, which are reasonable

Reply:
yeaaa, definitely worried it could go wild. 
just looked and saw that while we have `allow_shared_drives` as a bool, the tool didn't actually do anything with it so that should be fixed first